RecoBTag/MXNet/plugins/DeepBoostedJetTagsProducer.cc

```diff
@@ -31,8 +31,8 @@ struct MXBlockCache {
 struct PreprocessParams {
   struct VarInfo {
     VarInfo() {}
-    VarInfo(float median, float upper) :
-      center(median), norm_factor(upper==median ? 1 : 1./(upper-median)) {}
```

Comment:
I guess you changed the configuration parameters to obtain the same result as before, loading V01.
Numerically, though, there might be small differences because now you pass the (x,1/(x-y)) instead of (x,y).
[btw naively this looks strange... if upper is very close to median then 1/(upper-median) is >> 1 so it looks kind of discontinuous...]

Have you checked how much the output score is sensitive to this change?
Binned histogram tests with low statistics might not catch something like this...

Reply:
@peruzzim Yes, the conversion in https://github.com/cms-sw/cmssw/pull/26370/commits/5eb90b9806dd5fde155449eb8045d55b35ebfa09 actually introduced some very small differences in the outputs in rare cases, as I did the conversion in python where floating point numbers are assumed to be double precision while in the previous C++ code single precision was used. https://github.com/cms-sw/cmssw/pull/26370/commits/860b74c6514f87686adffbf9247e744c50a169c9 re-did the conversion in single precision, and now the outputs match the ones using previous version of code exactly (based on 1000 events).